dojo/tools/trivy/parser.py

```diff
@@ -173,7 +173,12 @@ def get_result_items(self, test, results, service_name=None, artifact_name=""):
                     if severity_source is not None and cvss is not None:
                         cvssclass = cvss.get(severity_source, None)
                         if cvssclass is not None:
-                            severity = self.convert_cvss_score(cvssclass.get("V3Score", None))
+                            v3score = cvssclass.get("V3Score", None)
+                            # Check if the severity score V3score not none before parsing it
```

Comment:
There is already a PR merged: https://github.com/DefectDojo/django-DefectDojo/pull/9268 
Did you take a look at the latest dev branch?